pkg/flags/flags.go

```diff
@@ -26,17 +26,18 @@ import (
 )
 
 const (
-	flagNamespaceName      = "namespace"
-	flagKubecofigName      = "kubeconfig"
-	flagFeatureName        = "feature"
-	flagAssessName         = "assess"
-	flagLabelsName         = "labels"
-	flagSkipLabelName      = "skip-labels"
-	flagSkipFeatureName    = "skip-features"
-	flagSkipAssessmentName = "skip-assessment"
-	flagParallelTestsName  = "parallel"
-	flagDryRunName         = "dry-run"
-	flagFailFast           = "fail-fast"
+	flagNamespaceName       = "namespace"
+	flagKubecofigName       = "kubeconfig"
+	flagFeatureName         = "feature"
+	flagAssessName          = "assess"
+	flagLabelsName          = "labels"
+	flagSkipLabelName       = "skip-labels"
+	flagSkipFeatureName     = "skip-features"
+	flagSkipAssessmentName  = "skip-assessment"
+	flagParallelTestsName   = "parallel"
+	flagDryRunName          = "dry-run"
+	flagFailFast            = "fail-fast"
+	flagIgnorePanicRecovery = "ignore-panic-recovery"
```

Comment:
I struggle with the name of the flag.
I think we should name the flag as it relates to the features of the framework, not as a function of how Go works.
What about `--disable-graceful-teardown` or `--force-finish`. That way it relates to how the framework functions.

Reply:
No worries. I do like the `--disable-graceful-teardown`. I will update it to this one.